tests/MongoDB.Driver.Tests/MongoCollectionImplTests.cs

```diff
@@ -1541,7 +1546,8 @@ private class ClassForDistinctWithArrayField
                 TextIndexVersion = 60,
                 Unique = true,
                 Version = 70,
-                Weights = weights
+                Weights = weights,
+                WildcardProjection = wildcardProjectionDefinition
```

Comment:
If this were an integration test it would fail when useWildcardIndex is false.

Reply:
Done